### PR TITLE
Added feedback on user creation fail

### PR DESCRIPTION
**Pull request by chhavip**
@shehbashahab  @zassmin  Added feedback for when user creation fails either due to unavailability of network or due to some internal error. Solves #427

### CODE REVIEW

File: app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java [hunk 1]
```diff
@@ -84,28 +86,33 @@ public void signUpUser(View view) {
...
+                                i.putExtra("userAns", userAns);
+                                startActivity(i);
+                                overridePendingTransition(R.anim.slide_in_right, R.anim.slide_out_left);
+                            } catch (Exception ex) {
+
+                            }
+                        } else {
+                            // TODO: Provide feedback and next steps to user when creation fails
+                            Snackbar.make(msnackbar, R.string.user_creation_failed, Snackbar.LENGTH_LONG).show();
+                            Log.d("User Creation failed", e.toString());
```
Comment (zassmin) [inline app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java]: Could this message change to "User information is invalid, please try again" and add it as a string resource?

**Comment by zassmin** [comment]
@chhavip looks good 👍! Just one comment and it'll be ready to merge!

**Comment by chhavip** [comment]
@zassmin The required change has been made at https://github.com/chhavip/WomenWhoCode/blob/user_creation_fail/app/src/main/res/values/strings.xml#L73 :)

**Comment by chhavip** [comment]
The required changes have been made @zassmin !

File: app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java [hunk 2]
```diff
@@ -116,6 +123,26 @@ public void goHome(View view) {
         intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
         startActivity(intent);
     }
+
+    public boolean isNetworkOnline() {
```
Comment (zassmin) [inline app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java]: @chhavip we have a singleton for this: https://github.com/WomenWhoCode/WomenWhoCode/blob/master/app/src/main/java/com/womenwhocode/womenwhocode/utils/NetworkConnectivityReceiver.java.

**Comment by zassmin** [comment]
@chhavip just about ready, during my initial review I missed pointing out that we have a singleton for much of what you wrote in `isNetworkOnline()`. Maybe try working with that? lmk.

**Comment by chhavip** [comment]
Will fix this soon

**Comment by chhavip** [comment]
@zassmin , done.

**Comment by zassmin** [comment]
@chhavip 🎉 👍 💯 (sorry about the delay, been traveling - it'd be fun to be able to do away messages on github.)

**Comment by chhavip** [comment]
@zassmin or have Slack deliver comments made on issues and PRs? ;-)

**Pull request merged**